aws/data_source_aws_cloudwatch_log_group.go

```diff
@@ -0,0 +1,63 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceAwsCloudwatchLogGroup() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAwsCloudwatchLogGroupRead,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"creation_time": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func dataSourceAwsCloudwatchLogGroupRead(d *schema.ResourceData, meta interface{}) error {
+	name := aws.String(d.Get("name").(string))
```

Comment:
Minor nitpick: usually its easier to work with the `name := d.Get("name").(string)` and only wrap it with `aws.String(name)` when creating the AWS SDK structs so you don't need to dereference the pointer every time. 👍